packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -99,6 +109,7 @@ SidePanel.defaultProps = {
 	reverse: false,
 	large: false,
 	dockable: true,
+	minimised: false,
```

Comment:
why do you not use !large instead?

Reply:
<img width="597" alt="image" src="https://user-images.githubusercontent.com/26482371/65252636-a3b87f00-daf9-11e9-8dcc-ebed4938b547.png">
large mode does not have the same effet than the new minimised mode

Reply:
ok so actually `minimised` is the same as the combination of `docked={true}` and `dockable={false}`

Reply:
Maybe you can just remove this props?

Reply:
Which props would you like to remove ?

Reply:
It's ok as it, it will be more understandable to apply guidelines